network/openvswitch_port.py

```diff
@@ -47,44 +54,188 @@
         default: 5
         description:
             - How long to wait for ovs-vswitchd to respond
+    iface-id:
+        required: false
```

Comment:
just update the code and push again, the PR will have an additional commit (or you can rebase and force push)